public/app/plugins/panel/table/transformers.ts

```diff
@@ -73,6 +73,56 @@ transformers['timeseries_to_columns'] = {
   }
 };
 
+transformers['grouping_to_rows_columns'] = {
+  description: 'Grouping to rows and columns',
+  getColumns: function() {
+    return [];
+  },
+  transform: function(data, panel, model) {
+    // Getting the columns
+    const points = new Map<string, Map<string, any>>();
+
+    const columns = new Set<string>();
+    const rows = new Set<string>();
+
+    for (let series of data) {
+      const columnsString = series.target.slice(
+        series.target.indexOf("{") + 1, -1);
+      const columnsArray = columnsString.split(", ");
+      const column = columnsArray[0].split(": ")[1];
+      const row = columnsArray[1].split(": ")[1];
+      columns.add(column);
+      rows.add(row);
+
+      if (!points.has(column)) {
+        points.set(column, new Map<string, any>());
+      }
+
+      // We take only the first measure reported (datapoints[0]) and
+      // We don't care about the timestamp of that measure (datapoints[0][0])
+      points.get(column).set(row, series.datapoints[0][0]);
+    }
+
+    // Removing the empty or 0-only rows and columns
+    const filteredRows = Array.from(rows.values()).filter(
+      r => points.has(r) && Array.from(points.get(r).values()).reduce(
+        (p, c) => p || c !== 0, false));
```

Comment:
I am removing here the empty and columns and rows (understanding by empty as something null or value 0) because it is fitting to our use case. Is it correct to have it like this upstream?